.github/workflows/ci.yml

```diff
@@ -28,9 +28,11 @@ jobs:
         run: npm clean-install
       - name: Lint
         run: npm run lint
-      - name: Test
-        run: npm test
-      - name: Coveralls
+      - name: Integration Tests
+        run: npm run test:integration
+      - name: Test Coverage
+        run: npm run test:coverage
```

Comment:
In theory the unit tests should be faster than the integration tests. Run those firsts so things can fail faster.

Reply:
This is correct in most scenarios. Currently that's not the case with Winston. In large part due to the stress tests defined in the `unit/transports` dir . I'm happy to make this change as ideally they should run faster (eventually). In reality I'm pretty sure all the `unit` tests are actually `integration` but I didn't want to be too heavy headed for the time being. Beyond that the more time I spent understanding the state of all current tests the more I wanted to take this further which as much as I like some of this "housekeeping", it's purely subjective and not directly impactful towards improving things.